configuration/config2/EnvValue.cpp

```diff
@@ -0,0 +1,114 @@
+/*##############################################################################
+
+HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems�.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+############################################################################## */
+
+#include "EnvValue.hpp"
+
+bool EnvValue::setValue(const std::string &value, bool force) 
+{ 
+	bool rc = true;
+    std::string oldValue = m_value;
```

Comment:
ok so m_value is THE value, and m_pCfgValue is the definition for that value right?
Idk, CfgValue might be good enough, but perhaps a slight name change will be more informative
how about p_valueDefinition ?

Reply:
Above comment wasn't addressed.